apps/src/labs/projects/ChannelsStore.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * This file contains the ChannelsStore interface and the local and remote implementations
+ * of the ChannelStore. A ChannelsStore manages the loading and saving of channels.
```

Comment:
Can we explain what local and remote mean in this context?  (Maybe not implementation, but at least the conceptual difference?). At first I thought it meant client and server code, until I remembered that we have a couple ways to save things.